resources/measures/hvac_nighttime_oa_controls/measure.rb

```diff
@@ -0,0 +1,206 @@
+# insert your copyright here
+
+# see the URL below for information on how to write OpenStudio measures
+# http://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/
+
+# start the measure
+class NighttimeOAControls < OpenStudio::Measure::ModelMeasure
+  # human readable name
+  def name
+    # Measure name should be the title case of the class name.
+    return 'Nighttime OA Controls'
+  end
+
+  # human readable description
+  def description
+    return 'This measure sets minimum outdoor airflow to zero during extended periods of no occupancy (nighttime). Fans cycle during these unoccupied periods to meet the thermostat setpoints.'
```

Comment:
I think we may want to change the naming from "nighttime" to "unoccupied periods", since this can include weekends and other unoccupied periods  as well.

Reply:
Good point! Done.